madmimiofficial.php

```diff
@@ -114,32 +132,37 @@ public function register_shortcode() {
 	}
 
 	public function register_widget() {
+
 		register_widget( 'Mad_Mimi_Form_Widget' );
+
 	}
 
 	public function enqueue() {
 
+		$suffix = SCRIPT_DEBUG ? '' : '.min';
+		$rtl    = ! is_rtl() ? '' : '-rtl';
+
 		// main JavaScript file
-		wp_enqueue_script( 'mimi-main', plugins_url( 'js/mimi.js', __FILE__ ), array( 'jquery' ), MADMIMI_VERSION, true );
+		wp_enqueue_script( 'mimi-main', plugins_url( "js/mimi{$suffix}.js", __FILE__ ), array( 'jquery' ), MADMIMI_VERSION, true );
 
 		// datepicker JavaScript file
-		wp_enqueue_script( 'function', plugins_url( 'js/function.js', __FILE__ ), array( 'jquery' ), MADMIMI_VERSION, true );
+		wp_enqueue_script( 'function', plugins_url( "js/function{$suffix}.js", __FILE__ ), array( 'jquery' ), MADMIMI_VERSION, true );
 
 		// JQuery-ui
-		wp_enqueue_script('jquery-ui', 'http://code.jquery.com/ui/1.11.4/jquery-ui.js', array('jquery'), '1.11.4');
+		wp_enqueue_script( 'jquery-ui', plugins_url( "js/jquery-ui{$suffix}.js", __FILE__ ), array( 'jquery' ), '1.11.4' );
```

Comment:
@EvanHerman IMO we should use the jQuery UI Datepicker script that is [bundled with WordPress](https://developer.wordpress.org/reference/functions/wp_enqueue_script/#default-scripts-included-and-registered-by-wordpress).

Can you swap it out and test?

```php
wp_enqueue_script( 'jquery-ui-datepicker' );
```